HelloWorldService/pom.xml

```diff
@@ -94,16 +94,6 @@
       <artifactId>swagger-integration</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>org.testng</groupId>
-      <artifactId>testng</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.assertj</groupId>
```

Comment:
I'd suggest we still use assertj, but we can use it where needed.  Looks like it was just an unused dependency here.

Reply:
That's correct - both testng and assertj are unused here.
We are still using `assertj` in the one test we have. I prefer it to the straight JUnit5 `asserts`.